### PR TITLE
kodi: enable dav1d in ffmpeg backend

### PR DESCRIPTION
**Pull request by zalaare**

### CODE REVIEW

**Comment by lrusak** [comment]
Not much point IMO. We should wait for dav1d decoder really. libaom is just the reference design and is sooooooooo slow

http://www.jbkempf.com/blog/post/2018/Introducing-dav1d
https://github.com/rbultje/ffmpeg/commit/3cdb0dc45c322d108ac15f59df915313ba6865e9

**Comment by zalaare** [comment]
It works for me now and since some of my personal encodes are av1, I'm using regardless.  dav1d can be replace libaom at any point in the future when it is ready.

**Comment by lrusak** [comment]
It's basically impossible to playback AV1 at full speed in software decoding. So I don't see any use for this currently.

**Comment by CvH** [comment]
http://www.jbkempf.com/blog/post/2018/dav1d-toward-the-first-release

**Comment by zalaare** [comment]
I did this again using dav1d.  Gotta clean up some of these commits and I can squash if need be.

**Comment by CvH** [comment]
@zalaare I guess something gone wrong :) maybe rebase ?

**Comment by zalaare** [comment]
> @zalaare I guess something gone wrong :) maybe rebase ?

Fixed.

File: packages/lang/nasm/package.mk [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+# Copyright (C) 2009-2016 Stephan Raue (stephan@openelec.tv)
+
```
[the remark this answers is not in the document]
Comment (zalaare) [reply packages/lang/nasm/package.mk]: I apologize for not understanding the github gui sometimes.  Is there something more I need to do for this License header change?  You want me to add our Team LibreELEC copyright to it?

**Comment by CvH** [comment]
@zalaare could you squash the commits into a single one per package ??

**Comment by zalaare** [comment]
> @zalaare could you squash the commits into a single one per package ??

Can do

**Comment by 5schatten** [comment]
tested with these samples https://www.elecard.com/videos and works fine though you'll still need a fast cpu for software decoding

**Pull request merged**